api/services/contact_fields_mapping_service.py

```diff
@@ -9,25 +9,25 @@ def set_init_data(self, jsonData):
         user_phone = helpers.fetch_by_key("urn", jsonData["contact"])
         self.user_phone = helpers.sanitize_phone_string(user_phone)
 
+    def contact_field_mappings():
+        return db.session.query(models.ContactFieldsMapping).all()
+
```

Comment:
it should be inside model with name `get_all_campaign_mapping`

Reply:
Added.